spa/src/ajax/portal-axios.ts

```diff
@@ -0,0 +1,15 @@
+import { PORTAL } from 'routes';
+import Axios from './axios';
+import { AxiosError } from 'axios';
+
+export default Axios;
+
+/**
+ * Overrides response interceptor from base Axios instance
+ */
+Axios.interceptors.response.use((success) => success, handleResponseError);
+
+function handleResponseError(error: AxiosError) {
+  if (error?.name === 'AuthenticationError') return window.location.assign(PORTAL.ENTRY);
+  return Promise.reject(error);
+}
```

Comment:
```suggestion
function handleResponseError(error: AxiosError) {
  if (error?.name === 'AuthenticationError') {
    window.location.assign(PORTAL.ENTRY);
  }
  
  return Promise.reject(error);
}
```

I'm fuzzy on whether JS code would continue to execute after `location.assign()` is called, but it seems like it's a good idea to always return the error.